api/scm/sync.go

```diff
@@ -124,7 +125,7 @@ func SyncRepo(c *gin.Context) {
 		}
 
 		// update webhook
-		err = scm.FromContext(c).Update(u, r, lastHook.GetWebhookID())
+		_, err = scm.FromContext(c).Update(u, r, lastHook.GetWebhookID())
```

Comment:
I say we do the same think at the repo level as we do the org level (i.e. change the repo to inactive if the webhook isn't found when attempting to update)